### PR TITLE
Add documentation and CSS for `bento-wordpress-embed` and `BentoWordPressEmbed`

### PR DESCRIPTION
**Pull request by caroqliu**
Documentation is needed for publishing on npm, and CSS is needed for providing layout stabilizing CSS (pre-upgrade)

### CODE REVIEW

File: extensions/amp-wordpress-embed/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  <style data-bento-boilerplate>
+    bento-wordpress-embed {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0/bento-wordpress-embed-1.0.js"></script>
+</head>
+<bento-wordpress-embed id="my-embed
```
Comment (alanorozco) [inline extensions/amp-wordpress-embed/1.0/README.md:54]:
```suggestion
<bento-wordpress-embed id="my-embed"
```

File: extensions/amp-wordpress-embed/1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-wordpress-embed
+```
+
+```javascript
```
Comment (alanorozco) [inline extensions/amp-wordpress-embed/1.0/README.md:125]: I don't know if this makes a difference on the Github parser and/or the amp.dev parser, but maybe it should be JSX?

```suggestion
```jsx
```

**Review by alanorozco — APPROVED** [review]

File: extensions/amp-wordpress-embed/1.0/README.md (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-wordpress-embed/1.0/README.md:54]: Nice catch, thank you

File: extensions/amp-wordpress-embed/1.0/README.md (hunk 2 above)
Comment (caroqliu) [reply extensions/amp-wordpress-embed/1.0/README.md]: I've always wondered the difference here too. I updated it in the PR but not finding any definitive guidance online.

**Pull request merged**